tldw_Server_API/app/api/v1/endpoints/media_embeddings.py

```
@@ -58,9 +58,10 @@
 
 def _user_embedding_config() -> dict[str, Any]:
     cfg = settings.get("EMBEDDING_CONFIG", {}).copy()
-    user_db_base_dir = settings.get("USER_DB_BASE_DIR")
-    if not user_db_base_dir:
+    try:
         user_db_base_dir = str(DatabasePaths.get_user_db_base_dir())
+    except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)